### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
OpenTracing docs

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3863,6 +3863,20 @@ Enable API Team Deletion
...
+
+**True**: A Jaeger client is instantiated and each HTTP request wraps App and Store in OpenTracing layers.
+Context is added to App and Store and is passed down the layer chain to create OpenTracing 'spans'.
+
+By default no method parameters are reported to OpenTracing to avoid leaking sensitive information. Only the name of the method is reported. 
+
+**False**: OpenTracing is not enabled.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableOpenTracing": false`` with options ``true`` and ``false``.                                                    |
```
Comment (amyblais) [inline source/administration/config-settings.rst:3877]: The table is not rendering

Comment (justinegeffen) [reply source/administration/config-settings.rst]: It should be fixed now.

**Comment by justinegeffen** [comment]
Thanks, @wiersgallak. I edited it locally to fix the table and edited the text slightly (which I should have done in GitHub). Changes are committed but happy to revert if necessary.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3863,6 +3863,20 @@ Enable API Team Deletion
 | This feature's ``config.json`` setting is ``"EnableAPITeamDeletion": false`` with options ``true`` and ``false``.                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable OpenTracing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: A Jaeger client is instantiated and each HTTP request wraps App and Store in OpenTracing layers.
+Context is added to App and Store and is passed down the layer chain to create OpenTracing 'spans'.
```
Comment (reflog) [inline source/administration/config-settings.rst:3870]: I'd phrase it a little bit differently:
"A [Jaeger](https://www.jaegertracing.io/) client is instantiated and is used to trace each HTTP request as it goes through App and Store layers."

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -3863,6 +3863,20 @@ Enable API Team Deletion
 | This feature's ``config.json`` setting is ``"EnableAPITeamDeletion": false`` with options ``true`` and ``false``.                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable OpenTracing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: A Jaeger client is instantiated and each HTTP request wraps App and Store in OpenTracing layers.
```
Comment (justinegeffen) [inline source/administration/config-settings.rst:3869]:
```suggestion
**True**: A Jaeger client is instantiated and is used to trace each HTTP request as it goes through App and Store layers.
```

**Pull request merged**